packages/shared/src/utils/validation/declarative-validation-rules.ts

```diff
@@ -111,7 +111,7 @@ const initPreBuildDVRs = () => ({
         message: form_error_messages.letter_symbol,
     },
     number: {
-        func: (value: string, opts: TOptions) => validNumber(value, opts),
+        func: (value: string, opts: TOptions, _values?: Record<string, string | boolean>) => validNumber(value, opts),
```

Comment:
@jim-deriv It doesn't seem like `_values` are used here. If we don't use it, we don't need to pass it here then. Do we? 🙂

Reply:
@maryia-binary, Yes it's needed. I explained a bit about why it's needed in the t_migration_discussing channel